templates/coordinator/_committee_list.html.twig

```diff
@@ -28,7 +28,7 @@
             <td>
                 {% for messages in app.flashes(["error_#{committee.id}"]) %}
                     {% for message in messages %}
-                        <div class="form__errors">
```

Comment:
Why are you changing this?

Reply:
because the class name definition in CSS is form__error

Reply:
I don't believe so. There is a `form__error` and a `form__errors`. Look for `__errors` and you'll find it.